imap_processing/tests/ultra/unit/test_ultra_l1b.py

```diff
@@ -6,18 +6,41 @@
 
 
 @pytest.fixture()
-def mock_data_dict():
+def mock_data_l1a_dict():
+    # Create sample data for the xarray Dataset
+    epoch = np.arange(
+        "2024-02-07T15:28:37", "2024-02-07T16:24:50", dtype="datetime64[s]"
+    )[:5]
```

Comment:
Can you make the array length-5 instead of creating extras and then truncating?
```suggestion
    epoch = np.arange(
        "2024-02-07T15:28:37", "2024-02-07T15:28:42", dtype="datetime64[s]"
    )
```